lib/gcloud/pubsub/project.rb

```diff
@@ -271,7 +266,7 @@ def topics options = {}
       #   The message data. (+String+)
       # +attributes+::
       #   Optional attributes for the message. (+Hash+)
-      # <code>attributes[:autocreate]</code>::
+      # <code>attributes[:autocreate+::
```

Comment:
In this one instance `:autocreate` is being pulled from the `attributes` hash. Because hashes and named parameters don't mix.